shotover-proxy/src/transforms/util/unordered_cluster_connection_pool.rs

```diff
@@ -143,20 +144,18 @@ async fn rx_process<C: CodecReadHalf>(
                     }
                     Err(e) => {
                         info!("Couldn't decode message from upstream host {:?}", e);
-                        return Err(anyhow!(
-                                "Couldn't decode message from upstream host {:?}",
-                                e
-                            ));
+                        return Err(anyhow!("Couldn't decode message from upstream host {:?}", e));
                     }
                 }
             },
             Some(original_request) = return_rx.recv() => {
-            if return_message_map.len() > 1 || return_channel_map.len() > 1 {
-                info!("message map {:?}", return_message_map);
-                info!("channel map {:?}", return_channel_map);
-            }
+                if return_message_map.len() > 1 || return_channel_map.len() > 1 {
+                    info!("message map {:?}", return_message_map);
```

Comment:
This or the other message map log is giving an absurd amount of log spam.

Reply:
Oh its because the len of a MultiMap never goes down.

Reply:
I'll remove them.